t/unit/utils/test_time.py

```diff
@@ -177,6 +174,81 @@ def test_remaining():
     next_run = now + rem_time
     assert next_run == next_actual_time
 
+    """
+    Case 4: DST check (ZoneInfo, timedelta)
+    Suppose start (which is last_run_time) is in PDT while next_run is in PST,
+    Check whether there is an hour added to the time between now and the `next_run`
+    In 2022, DST ends on Nov 6
+    """
+    start = datetime(day=6, month=11, year=2022, hour=1, minute=15, tzinfo=eastern_tz, fold=0)
+    now = datetime(day=6, month=11, year=2022, hour=1, minute=34, tzinfo=eastern_tz, fold=1)
+    ends_in = timedelta(minutes=80)
+    next_actual_time = datetime(day=6, month=11, year=2022, hour=1, minute=35, tzinfo=eastern_tz, fold=1)
+    assert start.tzname() == "EDT"
+    assert now.tzname() == "EST"
+    assert next_actual_time.tzname() == "EST"
+    rem_time = remaining(start, ends_in, now)
+    print(start + ends_in - now)
```

Comment:
I guess we can remove the print() func which might be used for debug purpose here?

Reply:
Removed the print() function